bin/download.js

```diff
@@ -38,5 +38,8 @@ request("/graphql")
       return;
     }
 
+    console.log("⌛  Generating schema.d.ts …");
+    await execa("npx", ["graphql-codegen"]);
+
```

Comment:
I think we need to separate the download from updating the `schema.d.ts` file. The `.d.ts` can change even if no update exists in GiHub's schema due to updates in one of the `@graphql-codegen/*` packages.

I would suggest to do the following:

1. Create a new `bin/generate-typescript.js` file
2. In scripts, rename the current `"update"` script to `"download"`
3. Add a new `"generate-typescript"` script
4. Add a new `"update"` script which does `"npm run -s download && npm run -s generate-typescript"`

Then add a new workflow file `.github/workflows/update-graphql-codegen.yml` which runs on

```yml
"on":
  push:
    branches:
      - dependabot/npm_and_yarn/graphql-codegen/*
```

and updates the existing pull request from dependabot if it exists, compare `.github/workflows/update-prettier.yml`